core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -623,16 +628,30 @@ class EMConfig {
     var algorithm = Algorithm.MIO
 
 
+   class enumAnnotationsExperimental(
+       var variableName: String,
+       var validValues : String,
+       var experimentalValues: String
+   )
+    //
+    val listAnnotationsExperimental = mutableListOf<enumAnnotationsExperimental>()
+     fun addAnnotationsEnumExperiment()
+     {
+         listAnnotationsExperimental.add(enumAnnotationsExperimental(variableName = "problemType", validValues =  "REST", experimentalValues = "WEB"))
+
+     }
+
     enum class ProblemType {
         REST,
-        @Experimental
         WEB
-    }
+       }
 
+    @Experimental
     @Cfg("The type of SUT we want to generate tests for, e.g., a RESTful API")
```

Comment:
remove `@Experimental`. the option `problemType` is not experimental, just the choice `WEB` is, for now